util/awsservice/constant.go

```diff
@@ -77,3 +79,27 @@ func init() {
 	CloudformationClient = cloudformation.NewFromConfig(awsCfg)
 	XrayClient = xray.NewFromConfig(awsCfg)
 }
+
+// ReconfigureAWSClients reconfigures the AWS clients using a new region.
+func ReconfigureAWSClients(region string) error {
```

Comment:
nit: this is basically the same thing as `init`. any reason to keep the implementation separate? also, it's a bit odd that if you want to use a region that not us-west-2, you have to create all the client for us-west-2 and then re-create them all for the desired region. is there something that prevents us from setting the correct region in `init` to start with? I'm not sure where this function is actually called from, so these may be naive questions

Reply:
I cleaned up the code a little, but the problem is that I want to retain the initial `init()` functionality due to other tests using this package, but we also should have the option to test in different regions, which my test will have the option for.

I could remove the `init()` function and make sure to call this in the other files that use `awsservice` instead, but that seems prone to making errors and breaking tests due to the amount of places it's used.

Reply:
Makes sense to me. Thanks for consolidating the logic